homeassistant/components/kodi/notify.py

```diff
@@ -64,7 +64,7 @@ async def async_get_service(
         _LOGGER.warning(
             "Kodi host name should no longer contain http:// See updated "
             "definitions here: "
-            "https://www.home-assistant.io/integrations/media_player.kodi/"
+            "https://www.home-assistant.io/integrations/kodi/"
```

Comment:
Would be nicer to get the url from the loader but this is OK as well

Reply:
What loader?

Reply:
https://github.com/home-assistant/core/blob/eb52943d276b0713961f9ec439c32b55b43f60ac/homeassistant/loader.py#L685

Reply:
Good to know, thanks. If you don't feel strongly about it I'll just leave this PR as-is - this was just a drive-by web UI PR and I've got other PRs I'd rather focus on

Reply:
It’s fine as-is